edge/pkg/common/util/config.go

```diff
@@ -91,7 +92,12 @@ func createPEMfile(path string, pemBlock pem.Block) error {
 		return err
 	}
 
-	defer file.Close()
+	defer func() {
+		err := file.Close()
+		if err != nil {
+			klog.Errorf("file path %v close error: %v", path, err)
```

Comment:
```suggestion
			klog.Errorf("failed to close file, path: %v, error: %v", path, err)
```
more clear as above? @gy95 @vincentgoat